guides/alerting/alerting.md

```diff
@@ -13,15 +13,20 @@ Some of the notified events:
 
 This feature is **disabled by default**.
 
-Provide `ENABLE_ALERTING=true` as an environment variable when starting Trento.
+There are two ways to enable and configure SMTP alerting:
 
-## Delivery and Recipient
-A notification needs to be _delivered to someone_ in _some way_.
+- via Environment Variables (env-vars)
+- via WebUI/RESTful API
 
-With alerting enabled some extra configuration is needed to define the recipient and the delivery mechanism.
-
-Currently **authenticated SMTP** is the **only supported delivery mechanism** for alert notifications.
+The env-vars have precedence. If _any_ of the following env-vars is
+set, then alerting configuration is _only_ made via env-vars:
+`ENABLE_ALERTING`, `ALERTING_SENDER`, `ALERTING_RECIPIENT`,
```

Comment:
Also consider updating this guide 
https://github.com/trento-project/web/blob/main/guides/development/environment_variables.md